src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -877,8 +877,8 @@ public class CaseData extends Et1CaseData {
     private List<ReferralTypeItem> referralCollection;
     @JsonProperty("referralHearingDetails")
     private String referralHearingDetails;
-    @JsonProperty("selectReferralToReply")
-    private DynamicFixedListType selectReferralToReply;
+    @JsonProperty("selectReferral")
+    private DynamicFixedListType selectReferral;
```

Comment:
just for future reference. In this case, it's 'fine' to change the JsonProperty as the field isn't in production but this shouldn't really be changed once it has been created as it can cause problems with the data